rfc6962-bis.xml

```diff
@@ -1201,6 +1201,9 @@ an accepted root certificate.
                     <t hangText="extra_data:">
                       The base64-encoded unsigned data pertaining to the log entry. In the case of an X509ChainEntry, this is the <spanx style="verb">certificate_chain</spanx>. In the case of a PrecertChainEntryV2, this is the whole <spanx style="verb">PrecertChainEntryV2</spanx>.
                     </t>
+                    <t hangText="signed_certificate_timestamp:">
+                      A Signed Certificate Timestamp for the entry, in the same format as in <xref target="sct"/>.
```

Comment:
add-(pre-)chain returns a broken-down SCT (i.e. each field as a separate parameter), whereas I think you're proposing here to put the whole SCT blob into "signed_certificate_timestamp".
How is this "the same format" ?

Note that "leaf_input" contains the base64-encoded MerkleTreeLeaf structure, which already contains the SCT's timestamp and extensions (in "timestamped_entry").

If we assume that the log ID is already known to the caller, then the only SCT fields missing from the existing get-entries specification are "sct_version" and "signature".
So how about adding "sct_version" and "signature" (although I'd call it "sct_signature" here), instead of "signed_certificate_timestamp" ?